Strategic-Initiatives.md

```diff
@@ -11,20 +11,125 @@ and have the support needed.
 
 # Current Initiatives
 
-| Initiative        | Champion                        | Links                                                            |
-|-------------------|---------------------------------|------------------------------------------------------------------|
-| http2             | James Snell                     |                                                                  |
-| Modules           | Myles Borins                    | https://github.com/nodejs/node-eps/blob/master/002-es-modules.md |
-| N-API             | Michael Dawson                  | https://github.com/nodejs/abi-stable-node                        |
-| OpenSSL Evolution | Rod Vagg                        | https://github.com/nodejs/TSC/issues/364                         |
-| Workers           | Anna Henningson                 | https://github.com/nodejs/worker                                 |
-| Error Messages    | James Snell                     | https://github.com/nodejs/node/issues/11273                      |
-| Core Promise APIs | James Snell                     |                                                                  |
-| TSC Governance    | Myles Borins                    | https://github.com/nodejs/TSC/issues/383                         |
-| New Streams APIs  | James Snell + Jeremiah Senkpiel | https://github.com/nodejs/node/pull/16414                        |
-| Moderation        | Rich Trott                      |                                                                  |
-| Async Hooks       | Trevor Norris                   |                                                                  |
-| V8 Currency       | Michaël Zasso                   |                                                                  |
+* [http2](#http2)
+* [Modules](#modules)
+* [N-API](#n-api)
+* [OpenSSL Evolution](#openssl-evolution)
+* [Workers](#workers)
+* [Error Messages](#error-messages)
+* [Core Promise APIs](#core-promise-apis)
+* [TSC Governance](#tsc-governance)
+* [New Streams APIs](#new-streams-apis)
+* [Moderation](#moderation)
+* [Async Hooks](#async-hooks)
+* [V8 Currency](#v8-currency)
+
+## http2
+
+### Champion
+
+[James Snell][jasnell] 
```

Comment:
I should be included as stakeholder here, and probably the rest of the the @nodejs/http2 team.

Reply:
I've added you as a stakeholder. I'd like to get explicit consent before adding individuals to the list.

Do you think we should add teams in here to?